shared_model/cryptography/blob.hpp

```diff
@@ -128,7 +104,7 @@ namespace shared_model {
       std::string hex_;
     };
 
-    static inline std::string   toBinaryString(const Blob &b) {
+    inline std::string toBinaryString(const Blob &b) {
```

Comment:
Can we put it inside .cpp?

Reply:
No: https://stackoverflow.com/a/9268589/1953079